app/src/main/res/layout/activity_main.xml

```diff
@@ -4,15 +4,21 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@drawable/main_background"
     tools:context=".MainActivity">
 
-    <TextView
-        android:layout_width="wrap_content"
+    <Toolbar
+        android:id="@+id/main_toolbar"
+        style="@style/ToolbarStyle"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+    <Button
+        android:id="@+id/button_parking_size"
```

Comment:
please use the suggested naming convention
https://jeroenmols.com/blog/2016/03/07/resourcenaming/

Reply:
Done, thanks !

Reply:
pls, use the suggested convention given in the on bording course